mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/AccountNumbersController.java

```diff
@@ -19,4 +19,11 @@ public final ResponseEntity<AccountNumbers> get(@PathVariable("id") String accou
     AccessorResponse<AccountNumbers> response = gateway().accounts().accountNumbers().get(accountId);
     return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
   }
+
+  @RequestMapping(value = "/users/{user_id}/accounts/{id}/account_number", method = RequestMethod.GET, produces = BaseController.MDX_MEDIA)
```

Comment:
I would prefer camel case for path variables such as `userId`. It's handled inconsistently in other controllers, but I believe camel case is more common than snake case.

Reply:
I am not particular, though camelcase is more consistent with `account_number` in the path itself.